build-system/sources.js

```diff
@@ -126,8 +127,16 @@ const CLOSURE_SRC_GLOBS = [
   '!node_modules/core-js/modules/library/**.js',
 ].concat(COMMON_GLOBS);
 
+let destination =
+  typeof argv.destination === 'string' ? argv.destination : './dist/';
+if (!destination.endsWith('/')) {
+  destination = `${destination}/`;
+}
+const OUTPUT_DIR = destination;
```

Comment:
Personally I find it hard to read when `OUTPUT_DIR` comes with the ending `/`. Can we always remove the ending `/` instead?